src/connectors/voice-search/__tests__/connectVoiceSearch-test.js

```diff
@@ -257,6 +258,44 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/voice-searc
     });
   });
 
+  describe('getRenderState', () => {
+    it('returns the render state', () => {
+      const { widget, helper } = getInitializedWidget();
+
+      const initOptions = createInitOptions({ state: helper.state, helper });
+
+      const renderState = widget.getRenderState({}, initOptions);
+
+      expect(renderState.voiceSearch).toEqual({
+        isBrowserSupported: true,
+        isListening: false,
+        toggleListening: expect.any(Function),
+        voiceListeningState: undefined,
+        widgetParams: {},
+      });
+    });
+  });
+
+  describe('getWidgetRenderState', () => {
+    it('returns the widget render state', () => {
```

Comment:
Can we add another test with a custom createVoiceSearchHelper so we can check the output?